### PR TITLE
Myhomework 5

### PR DESCRIPTION
**Pull request by Edaurd13**

### CODE REVIEW

File: ui/tests/UHC-mytest1.test.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+
+       const resultSignOut = page.locator('//div[contains(@class, "myAccount__title___3VN4o")]');
+       const resultSignOutText= await resultSignOut.textContent();
+       expect(resultSignOutText).toStrictEqual('My Account');
+
+
+      
+
+
```
Comment (vladimirchizhenkov) [inline ui/tests/UHC-mytest1.test.ts:96]: Use eslint please
yarn eslint --fix

File: ui/tests/UHC-mytest1.test.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+       const userHello = page.locator('//*[contains(@class,"myAccount__arrowIcon___2R-t_")]');
+       await userHello.hover();
+
+       const btnSignOut = page.locator('//ul[contains(@class, "myAccount__logged___35v2L")]/li[4]');
+       await btnSignOut.click();
+
+
+       const resultSignOut = page.locator('//div[contains(@class, "myAccount__title___3VN4o")]');
```
Comment (vladimirchizhenkov) [inline ui/tests/UHC-mytest1.test.ts:89]: Don't use selectors with hash!
Don't use xpath by index!

File: ui/tests/UHC-mytest1.test.ts [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        const fillFormEmail = page.locator('//*[text()= "Email"]');
+        const alphabet = "qwertyuiuolasdfghjklzxcvbnm";
+        let  res = "";
+          if(res.length <10){
+            res += alphabet[Math.floor(Math.random()* alphabet.length -1)]
+          }
+        const newEmail = res + (Math.round(Math.random()*10000) + "@gmail.com");
+        await fillFormEmail.type(newEmail);
+
+        // await page.locator('//*[text()= "Email"]').fill('test@test.com');
```
Comment (vladimirchizhenkov) [inline ui/tests/UHC-mytest1.test.ts:52]: Remove comments pls

File: ui/tests/UHC-mytest1.test.ts (hunk 1 above)
Comment (Edaurd13) [reply ui/tests/UHC-mytest1.test.ts:96]: Will be done

**Pull request merged**